src/main/scala/bazeltools/bsp4bazel/runner/BazelRunner.scala

```diff
@@ -31,36 +31,100 @@ import scala.util.Failure
 import scala.util.Success
 import scala.util.Success.apply
 import scala.util.Try
+import bazeltools.bsp4bazel.runner.BazelRunner.Command
 
 trait BazelRunner:
```

Comment:
Overall LGTM. Could you add Scaladoc comments on both `BazelRunner` and `BspBazelRunner` to differentiate the two?